lib/manager/components/version/FeedVersionNavigator.js

```diff
@@ -315,6 +310,7 @@ export default class FeedVersionNavigator extends Component<Props, State> {
               publishFeedVersion={publishFeedVersion}
               renameFeedVersion={renameFeedVersion}
               user={user}
+              validateGtfsPlusFeed={this.props.validateGtfsPlusFeed}
```

Comment:
Holy cow that's a lot of props. It might be nice to connect a few of the child components so all this data isn't passed through. Something for later.

Reply:
I'll create a ticket for this to revisit: #455